### PR TITLE
Switch relationships with path and label selector rather than group selector

### PR DESCRIPTION
**Pull request by algairim**
Switch relationships with path and label selector rather than whole relationship group, relationships can belong to sub-groups to be managed separately

Signed-off-by: Mykola Mandra <mykola.mandra@cloudsoftcorp.com>

### CODE REVIEW

File: ui-modules/blueprint-composer/app/components/providers/blueprint-service.provider.js [hunk 1]
```diff
@@ -93,8 +93,8 @@ function BlueprintService($log, $q, $sce, paletteApi, iconGenerator, dslService,
                     source: entity,
                     target: relation.entity,
                     label: relation.name,
-                    pathSelector: 'relation', // the CSS class for path
-                    labelSelector: 'relation-label', // the CSS class for label
+                    pathSelector: 'relation-selector', // the CSS class for path
+                    labelSelector: 'relation-selector', // the CSS class for label
```
Comment (iuliana) [inline ui-modules/blueprint-composer/app/components/providers/blueprint-service.provider.js:97]: `pathSelector` and `labelSelector` both  seem to have the same value. Is this intended?

Comment (iuliana) [reply ui-modules/blueprint-composer/app/components/providers/blueprint-service.provider.js:97]: Nevermind, I noticed there is the only one css class for this purpose.

**Review by iuliana — COMMENTED** [review]
LGTM

**Comment by iuliana** [comment]
Waiting for build to pass then this can be merged.

File: ui-modules/blueprint-composer/app/components/providers/blueprint-service.provider.js (hunk 1 above)
Comment (algairim) [reply ui-modules/blueprint-composer/app/components/providers/blueprint-service.provider.js:97]: Yes, selectors are the same for arcs and labels for Brooklyn relationships, however, they are can be different in the downstream versions when selecting label and arc can be managed differently.

**Pull request merged**